bin/build-plugin-zip.sh

```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 
+ZIP_FILE='woocommerce-admin.zip';
+
+while [ $# -gt 0 ]; do
+	if [[ $1 == '-f' || $1 == '--features' ]]; then
+		export WC_ADMIN_ADDITIONAL_FEATURES="$2"
+	fi
+	if [[ $1 == '-s' || $1 == '--slug' ]]; then
+		ZIP_FILE="woocommerce-admin-$2.zip";
+	fi
+	shift
+done
```

Comment:
Ah ok, so the slug can be anything, ie `onboarding-test-wpcom`, but isn't required. Thats nice 👍

Reply:
Looks like the error reported here, https://github.com/woocommerce/woocommerce-admin/issues/3056

Reply:
That was indeed the same error @psealock. I've rebased this, and made sure to test the zip with that option cleared out. Should be fixed now.